styles/styles.css

```diff
@@ -0,0 +1,199 @@
+@import url("./fonts.css");
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  margin-bottom: 120px;
+  padding: 120px 104px 122px;
+  width: 1520px;
+}
+
+.title-block {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.title {
+  text-align: center;
+  color: #383838;
+  font-size: 40px;
+  font-weight: 500;
+}
+
+.line {
+  background-color: #f5bd41;
+  border: 2px solid #f5bd41;
+  width: 100px;
+  height: 1px;
+  align-self: center;
+  margin-top: 20px;
+}
+
+.row-quality {
+  display: flex;
+  margin-top: 80px;
+  width: 1232px;
+  height: 183px;
+}
+
+.quality-col {
+  width: 25%;
+  text-align: center;
+  color: #383838;
+  font-size: 24px;
+}
+
+.quality-title {
+  margin-top: 48px;
+  font-weight: 400;
+  line-height: 28px;
+  font-size: 24px;
+}
+
+.footer {
+  background-color: #10367c;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 120px 0 200px 104px;
+  color: #ffffff;
+  width: 1520px;
+}
+
+.colon-foot {
+  width: 33.3333%;
+}
+
+.colontitul {
+  margin-top: 64px;
+  font-size: 14px;
+}
+
+.text-foot {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 47px;
+}
+
+.text {
+  font-weight: 500;
+}
+
+.trip-house {
+  height: 900px;
+  width: 1520px;
+  background-image: url(".././images/fon.jpg");
+  background-repeat: no-repeat;
+  padding: 51px 104px 120px 104px;
+}
+
+.header-trip {
+  display: flex;
+  justify-content: space-between;
+  row-gap: 684px;
```

Comment:
Be afraid of such big gaps/paddings/margins. It should kill the flexibility and adaptivity. Anyway, it doesn't work here.

Reply:
Ok.